### DISCUSSION TITLE
提一个关于登录二维码小小的建议

### DISCUSSION THREAD

**Discussion by ChinaPhusun** [Mrs4s/go-cqhttp#1112] [Ideas]
建议把登录二维码缩小一些,,,目前在多个系统测试,打开cmd默认窗口大小情况下获取到二维码被遮挡一部分无法扫描,,,再放大窗口的话二维码就全黑了,,,,,目前正常的流程只能是打开cmd窗口,先放大窗口,,再运行go-cqhttp,获取到的二维码才能正常扫描.

如果把二维码缩小了,则可以免去放大cmd窗口再运行go-cqhttp这个步骤,,,本人认为虽然这只是一个小小的改进,,但却增强了不少用户体验,任何产品做到极致都体现在细微之处,,所以望予采纳!
![1](https://user-images.githubusercontent.com/38497378/137461979-9df029d6-f8f0-4af9-b6b8-97509a911883.jpg)
![2](https://user-images.githubusercontent.com/38497378/137461983-ea5f8fb8-3f33-443e-b9b4-878f587a7ea9.jpg)

**Comment by wdvxdr1123** [comment]
你可以直接打开qrcode.jpg文件

**Reply by ChinaPhusun** [reply]
> 你可以直接打开qrcode.jpg文件

我知道是可以打开code.jpg文件扫描,,,但是还是多了步骤,,,如果在cmd里把二维码缩小,则可以省去这些步骤了,岂不更方便一点.

**Reply by wdvxdr1123** [reply]
目前打印二维码是通过打印两个空格代表一个像素点，无法实现缩小
